### ISSUE TITLE
Relicense Ganeti under the BSD License

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#992]
Originally reported of Google Code with ID 936.

```
As discussed with contributors and at GanetiCON the codebase is to be relicensed under the BSD license (2-clause). Please implement the change, in whichever release we see fit. (2.10 onwards?)

Thanks,

Guido

```
Originally added on 2014-09-08 17:52:28 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Since the legal aspects are resolved (thanks Guido) the only task
remaining is to change the license text. Being on-duty this week,
I'll take care of this.
```
Originally added on 2014-09-09 09:58:53 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
For the record: the new license text will now read (with the
year(s) adapted accordingly). Guido, can you please confirm that
this correct.


Copyright (C) 2014 Google Inc.
All rights reserved.

Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions are
met:

1. Redistributions of source code must retain the above copyright notice,
this list of conditions and the following disclaimer.

2. Redistributions in binary form must reproduce the above copyright
notice, this list of conditions and the following disclaimer in the
documentation and/or other materials provided with the distribution.

THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED
TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

```
Originally added on 2014-09-09 12:24:21 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
The above text is approved.

Thanks,

Guido

```
Originally added on 2014-09-11 16:29:03 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
The license text has been updated everywhere accordingly.
```
Originally added on 2014-09-15 16:14:38 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.11.6
```
Originally added on 2014-09-22 15:56:40 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.11.6
```
Originally added on 2014-09-22 15:57:10 +0000 UTC.

Changed State: Released
